app/models/tour.rb

```diff
@@ -1,3 +1,10 @@
 class Tour < ApplicationRecord
   belongs_to :user
+
+  validates :name, presence: true
+  validates :city, presence: true
+  validates :starting_point, presence: true
+  validates :description, presence: true, length: { minimum: 100 }
```

Comment:
can you also add a maximum number of characters, please?

Reply:
Done!